### PR TITLE
Check local UDMI version against upstream

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: bin/sequencer [hunk 1]
```diff
@@ -40,4 +40,12 @@ echo
 # Sort by test name (6th field)
 fgrep 'RESULT ' $TEST_LOG | sort -k 6 | tee /tmp/sequencer.out
 
+<<<<<<< HEAD
```
Comment (johnrandolph) [inline bin/sequencer:43]: Is this a hereto construction? I don't see ending HEAD?

File: bin/sequencer [hunk 2]
```diff
@@ -40,4 +40,12 @@ echo
...
+echo Running tools version `(cd $ROOT_DIR; git describe)`
+
+echo SUMMARY `fgrep 'RESULT ' $TEST_LOG | awk '{print $3, $2}' | sort | awk '{print $2}'`
+
+bin/check_version
+=======
 echo SUMMARY `awk '{print $5}' < /tmp/sequencer.out`
+>>>>>>> master
```
Comment (johnrandolph) [inline bin/sequencer:51]: Similar confusion?

Comment (grafnu) [reply bin/sequencer]: leftover from a botched merge

File: bin/check_version [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+#
+# This script intentionally runs silent if there is no version mismatch found.
+#
+
+ROOT_DIR=$(dirname $0)/..
+cd $ROOT_DIR
+
+branch=master
+origin=`git remote -v | fgrep faucetsdn/udmi | awk '{print $1}' | uniq`
```
Comment (johnrandolph) [inline bin/check_version:10]: TBH I'm a little surprised that we can't do tihs a little more explicitly with git remote, but, at a quick glance it seems not. No problem for now though.

**Review by johnrandolph — APPROVED** [review]

**Pull request merged**